docs/agile-process.md

```diff
@@ -13,7 +13,11 @@ There is a [high-level roadmap](https://github.com/GSA-TTS/FAC/blob/main/docs/pr
  
 **Ceremonies** - The goal is for new work from the backlog to enter the sprint at the start of the sprint and have agreement from the team that it's realistic to get the body of work done by the end of the sprint. In practice, items are sometimes removed and added to the sprint while it's ongoing. When this happens, team members are expected to communicate the up-scope or down-scope of the sprint to the rest of the team. Items that the team agrees to take on from the outset are currently given a sprint milestone, which communicates what the team anticipated was possible.
  
-**Labeling issues** - In addition to the "story", "design" and "engineering" labels, the "needs refinement" label is a very important label to denote whether an issue needs to be better clarified. The goal is to never start a sprint with an issue labeled "needs refinement" in it. There are a bunch of other labels that are rarely used and could probably be deleted.
+**Labeling issues** - In addition to the "story", "design" and "engineering" labels, the "needs refinement" label is a very important label to denote whether an issue needs to be better clarified. There are a bunch of other labels that are rarely used and could probably be deleted. These two carry special significance since we try not to actively work on an issue until these labels have been removed: 
+
+* “Needs refinement” - need requirements a group decision (design, engineering, product) to move forward
```

Comment:
> “Needs refinement” - need requirements a group decision (design, engineering, product) to move forward

I wonder if "needs decision" would be more clear?

Reply:
Updated.